client/client.go

```diff
@@ -22,6 +22,7 @@ type Client interface {
 	GetIndices(ctx context.Context, indexPatterns []string) ([]byte, error)
 	NewBulkIndexer(context.Context) error
 	UpdateAliases(ctx context.Context, alias string, removeIndices, addIndices []string) error
+	MultiSearch(ctx context.Context, indices []string, document []byte) error
```

Comment:
What is the "document" that is being supplied? The body of the MultiSearch request is a collection of header/query pairs of newline delimited JSON documents. Having a function signiture like this is exposing underlying elasticsearch implementation to the consumer of this library. The method signature should take a collection of search query structs as a parameter so is clearer what it is doing.

Reply:
Fixed in update 1.